packages/fuels-core/src/types/transaction_builders.rs

```diff
@@ -636,10 +671,10 @@ impl CreateTransactionBuilder {
         self,
         mut base_offset: usize,
         chain_id: &ChainId,
-        network_min_gas_price: u64,
+        provider: &impl DryRunner,
```

Comment:
`DryRunner` has a blanket implementation for `&T`. As such it is more flexible to say `impl DryRunner` -- we accept both by value and by reference since both satisfy `impl DryRunner`.